internal/models.go

```diff
@@ -0,0 +1,22 @@
+package internal
+
+import (
+	"time"
+)
+
+type Account struct {
+	AccountId string `json:"account-id"`
+	Name      string `json:"name"`
+	LastName  string `json:"lastname"`
+}
+
+type Transaction struct {
```

Comment:
It is better to keep the same pattern for the json attributes. In some areas, you use kebab case in others you use camel case. Choose one and keep standard for all our responses